ecr/fetcher.go

```diff
@@ -65,7 +66,8 @@ func (f *ecrFetcher) Fetch(ctx context.Context, desc ocispec.Descriptor) (io.Rea
 		ocispec.MediaTypeImageLayer,
 		ocispec.MediaTypeImageConfig:
 		return f.fetchLayer(ctx, desc)
-	case images.MediaTypeDockerSchema2LayerForeign,
+	case
+		images.MediaTypeDockerSchema2LayerForeign,
```

Comment:
Can you move this into the commit where you're modifying the other media types?

Reply:
Certainly, the latest push should reflect this.